src/app/helpers/backbone/node-manager.js

```diff
@@ -0,0 +1,116 @@
+import {Region} from './view';
+
+export default class NodeManager {
+
+    constructor(model) {
+        this.model = model;
+        this.managedNodes = {};
+        this.bindings = {};
+    }
+
+    _addNode(node) {
+        let key = node.dataset.manager;
+
+        if (typeof key === 'undefined') {
+            return;
+        }
+
+        if (!(key in this.managedNodes)) {
+            this.managedNodes[key] = [];
+        }
+        this.managedNodes[key].push(node);
+    }
+
+    updateNodes(rootNode) {
+        if (!rootNode) {
+            return;
+        }
+        this.managedNodes = {};
+
+        this._addNode(rootNode);
+        for (let node of rootNode.querySelectorAll('[data-manager]')) {
+            this._addNode(node);
+        }
+
+        for (let key in this.managedNodes) {
+            if (!(key in this.bindings)) {
+                window.console.warn('No binding for:', key);
```

Comment:
I think it's best to avoid console statements in the code.  I'm assuming you made it `window.console.warn` instead of `console.warn` to get around ESLint throwing an error?